src/NzbDrone.Core/Extras/ExtraService.cs

```diff
@@ -69,12 +70,14 @@ private void ImportExtraFiles(LocalEpisode localEpisode, EpisodeFile episodeFile
             var files = _diskProvider.GetFiles(sourceFolder, SearchOption.TopDirectoryOnly);
 
             var wantedExtensions = _configService.ExtraFileExtensions.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
-                                                                     .Select(e => e.Trim(' ', '.'))
+                                                                     .Select(e => e.Trim(' ', '.').Insert(0, "."))
                                                                      .ToList();
+            var wantSubs = wantedExtensions.Any(e => SubtitleFileExtensions.Extensions.Contains(e));
```

Comment:
`wantSubs` , `hasSub` and other subtitle specific logic doesn't belong in this file.